packages/store/ts/library/render-solidity/renderTable.ts

```diff
@@ -129,6 +130,15 @@ library ${libraryName} {
     ])});
   }
 
+  /** Encode keys as a bytes32 array using this table's schema */
+  function encodeKey(${renderArguments([_typedKeyArgs])}) internal pure returns (bytes32[] memory _primaryKeys) {
```

Comment:
thinking we will want to consolidate our language around "key" vs "primary key" vs "key tuple" and wondering if we wanna just go ahead and name this `encodeKeyTuple` here

Reply:
Agreed - I will use `encodeKeyTuple`. I much prefer "key tuple" as we represent primary keys in many ways. 

To be clear I don't think we should rename the `_primaryKeys` vars in this PR (not that you are proposing that).

Reply:
yep! agreed